src/app/organization/bulk-import/bulk-import.component.scss

```
@@ -5,3 +5,140 @@
+          color: var(--md-sys-color-on-surface, #1a1c1e);
+          line-height: 1.2;
+        }
+
+        .menu-explanation {
+          color: var(--md-sys-color-on-surface-variant, #44474e);
+          font-size: 13px;
+          margin-top: 6px;
+          line-height: 1.4;
+          opacity: 0.8;
+          white-space: normal;
+          word-break: break-word;
```

Stylelint (17.3.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)